trusted_applet/main.go

```diff
@@ -78,6 +79,20 @@ var (
 	persistence *storage.SlotPersistence
 )
 
+var (
+	doOnce                       sync.Once
+	counterWitnessStarted        monitoring.Counter
+	counterFirmwareUpdateAttempt monitoring.Counter
+)
+
+func initMetrics() {
+	doOnce.Do(func() {
+		mf := monitoring.GetMetricFactory()
+		counterWitnessStarted = mf.NewCounter("witness_started", "Number of times the witness was started")
```

Comment:
Should we call this `witness_restarts`? I think it's mainly going to count the number of restarts due to DHCP lease expiries, if the device crashes and restarts I suspect this'll continue to show the same number to prometheus and it won't notice, unless there's some clever trickery behind the scenes?

Reply:
As discussed we'll keep this name as it is.